ansible/roles/postgresql/tasks/postgis.yml

```diff
@@ -14,18 +14,17 @@
     - postgresql
     - postgis
 
-# TODO Seems redundant given this occurs in debian.yml
-# - name: Install PostGIS if required
-#   apt:
-#     pkg: "{{ item }}"
-#     state: present
-#     update_cache: yes
-#   when: postgis_version is defined
-#   with_items:
-#     - "postgresql-{{pg_version}}-postgis-{{postgis_version}}"
-#   tags:
-#     - postgresql
-#     - postgis
+- name: Install PostGIS if required
```

Comment:
I am not sure the purposes of those three files. I noticed the installation of postgres exists in two places, but I am afraid of removing one of them, in case other projects need